ui/guest/src/controls/rte.ts

```diff
@@ -113,7 +113,7 @@ export function initTinyMCE(
         function save() {
           const content = getContent();
           if (changed) {
-            contentController.updateField(record.modelId, field.id, record.index, content);
+            contentController.updateField(record.modelId, record.fieldId[0], record.index, content);
```

Comment:
@rart the problem in here was that for example, when having a repeating group like:
`sections_o: [
     section_html,
     section_html, ...
]`
`field.id` would be `section_html`, but we need the full id (`record.fieldId` has the full value `sections_o.section_html`).